### ISSUE TITLE
Error using Cordova Plugin

### ISSUE THREAD

**Issue by altinselimi** [janeasystems/nodejs-mobile#146]
- Project: nodejs-mobile-cordova
- Version: 0.2.2
- Mobile device: simulator
- Mobile OS and version: iOS 12.1
- Development Node.js: v8.12.0
- Development platform: MacOS Mojave

Hello, 

I'm trying to use **cordova-sqlite-storage** inside my nodejs-project, in order to use **typeorm** library and initialize a database. 
I have installed **cordova-sqlite-storage** in the root project using `cordova add plugin cordova-sqlite-storage`, and its properly sitting in my package.json. But unfortunately when running on the simulator, this error pops up:
```Error:  { DriverPackageNotInstalledError: Cordova-SQLite package has not been found installed. Try to install it: npm install cordova-sqlite-storage --save```

I've tried to install it using npm inside the nodejs-project, and also requiring it at the top of the file, but it doesn't seem to make any difference.

Any help would be appreciated 🙌🏻

**Comment by jaimecbernardo** [comment]
Hi @altinselimi ,
Looks like cordova-sqlite-storage is a Cordova plugin, and not something to use inside a node runtime. This might be why it isn't working in the node runtime.

**Comment by altinselimi** [comment]
Yeah I was not sure whether it should work or shouldn't as it is not explicitly mentioned in the docs. All I saw was 'crodova-bridge' was being required when code was being executed in the phone, so I assumed I can use cordova plugins too. 😬
